src/SFML/Audio/resource.rc.in

```diff
@@ -0,0 +1,32 @@
+#include "winresrc.h"
```

Comment:
You don't need a resource file to configure per module. You can have a single one configured differently for each module

Reply:
What relevance does this comment have to the `#include` statement, or are you just trying to attach it somewhere?

Reply:
Indeed it was related to the whole file